### PR TITLE
allow users to provide relative path in planckton flow root directory

### LINKED ISSUE #49: Make it easier to provide input files with planckton-flow
This may need changing in planckton-flow, but in order to provide an input compound when running through a container, the full path needs to be provided.

### PR DESCRIPTION
**Pull request by jennyfothergill**
fixes https://github.com/cmelab/planckton/issues/49

- provided path can be absolute path or relative path from planckton-flow root directory
- More descriptive error message

### CODE REVIEW

File: src/project.py [hunk 1]
```diff
@@ -85,7 +86,24 @@ def get_paths(key):
...
+        if path.isfile(key):
+            print(f"Using {key} for structure")
+            return key
+        elif path.isfile(file_path):
+            print(f"Using {file_path} for structure")
+            return file_path
+        raise FileNotFoundError(
+            "Please provide either a path to a file (the absolute path or the "
+            "relative path in the planckton-flow root directory) or a key to "
+            f"the COMOUND dictionary: {COMPOUND.keys()}\n"
```
Comment (JimmyRushing) [inline src/project.py]: did you mean compound silly goose?

Comment (JimmyRushing) [inline src/project.py]:
```suggestion
            f"the COMPOUND dictionary: {COMPOUND.keys()}\n"
```

**Comment by erjank** [comment]
F the comound dictionary

**Review by JimmyRushing — CHANGES_REQUESTED** [review]
It thought the relative path was "y6.mol2 /Users/jamesrushing/y6.mol2"

**Comment by jennyfothergill** [comment]
> It thought the relative path was "y6.mol2 /Users/jamesrushing/y6.mol2"

Thanks for reviewing, @JimmyRushing! I had tested this before using signac operations but something about directly running the test_local file was different. 🤔 
Anyway, it's fixed now!

**Review by JimmyRushing — APPROVED** [review]
I ran with the absolute path and then i ran with only the relative path and it located the file both times.

**Pull request merged**